packaging/windows/buildpackage.sh

```diff
@@ -29,23 +29,32 @@ elif [[ ${TAG} == playtest* ]]; then
 	SUFFIX=" (playtest)"
 fi
 
+if command -v curl >/dev/null 2>&1; then
+	curl -s -L -O https://github.com/electron/rcedit/releases/download/v1.1.1/rcedit-x64.exe || exit 3
+else
+	wget -cq https://github.com/electron/rcedit/releases/download/v1.1.1/rcedit-x64.exe || exit 3
+fi
+
 function makelauncher()
 {
 	LAUNCHER_NAME="${1}"
 	DISPLAY_NAME="${2}"
 	MOD_ID="${3}"
 	PLATFORM="${4}"
 
+	sed "s|DISPLAY_NAME|${DISPLAY_NAME}|" ../../OpenRA.WindowsLauncher/ModConfig.cs.in | sed "s|MOD_ID|${MOD_ID}|" | sed "s|FAQ_URL|${FAQ_URL}|" > ../../OpenRA.WindowsLauncher/ModConfig.cs
 	# Create multi-resolution icon
 	convert "${ARTWORK_DIR}/${MOD_ID}_16x16.png" "${ARTWORK_DIR}/${MOD_ID}_24x24.png" "${ARTWORK_DIR}/${MOD_ID}_32x32.png" "${ARTWORK_DIR}/${MOD_ID}_48x48.png" "${ARTWORK_DIR}/${MOD_ID}_256x256.png" "${BUILTDIR}/${MOD_ID}.ico"
 
-	# Create mod-specific launcher
-	msbuild -t:Build "${SRCDIR}/OpenRA.WindowsLauncher/OpenRA.WindowsLauncher.csproj" -restore -p:Configuration=Release -p:TargetPlatform="${PLATFORM}" -p:LauncherName="${LAUNCHER_NAME}" -p:LauncherIcon="${BUILTDIR}/${MOD_ID}.ico" -p:ModID="${MOD_ID}" -p:DisplayName="${DISPLAY_NAME}" -p:FaqUrl="${FAQ_URL}"
-	cp "${SRCDIR}/bin/${LAUNCHER_NAME}.exe" "${BUILTDIR}"
+	dotnet publish ../../OpenRA.WindowsLauncher/OpenRA.WindowsLauncher.csproj -r ${PLATFORM} -p:LauncherIcon="${BUILTDIR}/${MOD_ID}.ico" -p:LauncherName=${LAUNCHER_NAME} -p:TargetPlatform=${PLATFORM} p:ModID="${MOD_ID}" -p:DisplayName="${DISPLAY_NAME}" -p:FaqUrl="${FAQ_URL}" -o ${LAUNCHERBUILTDIR}
```

Comment:
Can we `-o ${BUILTDIR}` like we do for the main engine and avoid the mv?

Reply:
We may also need to remove the -p:LauncherIcon?

Reply:
why?

Reply:
Because it doesn't work 😁 
I don't remember whether it gives an error or just silently does nothing, but in either case its redundant since we are adding the icon using rcedit below.

Reply:
It would be a good idea to add a comment explaining this too - then at some point in the future once .NET has fixed this we will be able to remember the context and migrate back.